src/main/kotlin/rx/lang/kotlin/observables.kt

```diff
@@ -1,12 +1,14 @@
 package rx.lang.kotlin
 
+import rx.AsyncEmitter
 import rx.Observable
 import rx.Subscriber
 import rx.Subscription
 import rx.observables.BlockingObservable
 
 fun <T> emptyObservable() : Observable<T> = Observable.empty()
 fun <T> observable(body : (s : Subscriber<in T>) -> Unit) : Observable<T> = Observable.create(body)
+fun <T> fromAsync(body: (s: AsyncEmitter<in T>) -> Unit, mode: AsyncEmitter.BackpressureMode): Observable<T> = Observable.fromAsync<T>(body, mode)
```

Comment:
I know you prefer static imports but then I have to remember this import rule because there is no help from the IDE to add this import.

With this method I have autocompletion from my IDE when I begin to enter `fromAs...`. It's just to make my work easier.